.gitmodules

```diff
@@ -1,3 +1,5 @@
 [submodule "jsdoc-template"]
 	path = jsdoc-template
-	url = https://github.com/andreGarvin/jsdoc-template
+	url = git@github.com:openfin/jsdoc-template.git
```

Comment:
should it use `https://github.com/openfin/jsdoc-template` just like [here](https://github.com/openfin/javascript-adapter/blob/develop/.gitmodules) to prevent auth problems?